metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -1065,6 +1067,22 @@ def parse_sql_lineage(
             default_schema=default_schema,
         )
 
+    def create_tag(self, tag_name: str) -> Dict[Any, Any]:
```

Comment:
this should return just the urn, and not a dict

Reply:
done